docs/conf.py

```diff
@@ -0,0 +1,56 @@
+"""Sphinx documentation generator configuration file.
+
+The full set of configuration options is listed on the Sphinx website:
+http://sphinx-doc.org/config.html
+
+"""
+import sys
+import os
+
+
+# Add the AirGun root directory to the system path. This allows references
+# such as :mod:`airgun.browser` to be processed correctly.
+sys.path.insert(
+    0,
+    os.path.abspath(os.path.join(
+        os.path.dirname(__file__),
+        os.path.pardir
+    ))
+)
+
+# Project Information ---------------------------------------------------------
+
+project = 'AirGun'
+copyright = '2018, Andriy Balakhtar'
```

Comment:
Imo this copyright is unfair, i'm not the only one who works and contributes to the project. Maybe `RedHat QE Team` (since not only satellite QEs contribute here?) or smth? @omaciel any suggestions?

Reply:
IMO you have started with the project and IMO this does not say that you have created it all, but that you are the one holding copyright rights. But IMNAL.